src/components/Table/hoc/withTableSelection/withTableSelection.tsx

```diff
@@ -50,6 +51,7 @@ export function withTableSelection<I extends TableDataItem, E extends {} = {}>(
                     columns={this.enhanceColumns(columns)}
                     onRowClick={this.enhanceOnRowClick(onRowClick)}
                     getRowClassNames={this.enhanceGetRowClassNames(getRowClassNames)}
```

Comment:
If we pass `getRowDescriptor`, it is no need to pass `getRowClassNames`.

Reply:
but we still want support old props for a while

Reply:
Yeah, but it's internal <TableComponent> call. So we may take `getRowClassNames` into account inside `enhanceGetRowDescriptor`.